src/cmd-remote-build-container

```diff
@@ -230,9 +238,21 @@ Examples:
     parser.add_argument(
         '--git-sub-dir', default='', required=False,
         help='Git sub directory to use for container build')
+    parser.add_argument(
+        '--label', dest="labels", default=[], action='append',
+        required=False, help='Add image label(s)')
+    parser.add_argument(
+        '--mount-host-ca-certs', required=False, action='store_true',
+        help='Mount the CA certificate from the remote host')
     parser.add_argument(
         '--repo', default='localhost', required=False,
         help='Registry repository')
+    parser.add_argument(
+        '--secret', required=False,
+        help='Provide a local secret for remote access')
+    parser.add_argument(
+        '--security-opt', required=False,
+        help='Set SELinux labels')
```

Comment:
I think either we keep this called `--security-opt` but then it takes a value that follows the same semantics as the real `--security-opt` (we'd just pass it through as is), or we rename this to e.g. `--no-selabel` and translate it to `--security-opt label=disable`.